src/styles/blocks/header.scss

```diff
@@ -19,6 +19,7 @@
     display: flex;
     margin: 0;
     padding: 0;
+    transition: 300ms;
```

Comment:
add transition-property. By default it's `"all"`. You should specify what exactly you would like to change